package.json

```diff
@@ -16,7 +16,7 @@
     "test:react": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
     "electron-dev": "concurrently \"BROWSER=none npm start\" \"wait-on http://localhost:3000 && electron --enable-sandbox .\"",
-    "electron-pack": "build --em.main=build/electron.js",
+    "electron-pack": "build --c.extraMetadata.main=build/electron.js",
```

Comment:
So this fixes the build?

Reply:
Both changes are part of the fix. However, the line this change replaces is the reason Travis isn't uploading any files on releases